### ISSUE TITLE
document the row type returned by gproc:table/N

### ISSUE THREAD

**Issue by qwfy** [uwiger/gproc#134]
Currently, one cannot know (without trying or digging the source code) what's the type of `Row` in `qlc:e(qlc:q([Row || Row <- gproc:table()]))`, should this be documented?

**Comment by uwiger** [comment]
I agree that it should be clarified. Currently, one can infer it from [the documentation of `gproc:select/2`](https://github.com/uwiger/gproc/blob/master/doc/gproc.md#select2).

That is: gproc defines a logical view of the registry as `[{Key, Pid, Value}]`.
